RecoEcal/EgammaClusterAlgos/interface/SCEnergyCorrectorSemiParm.h

```diff
@@ -49,6 +51,13 @@ class SCEnergyCorrectorSemiParm {
   edm::ESHandle<CaloTopology> calotopo_;
   edm::ESHandle<CaloGeometry> calogeom_;
 
+  edm::ESGetToken<CaloTopology, CaloTopologyRecord> tokenCaloTopo_;
+  edm::ESGetToken<CaloGeometry, CaloGeometryRecord> tokenCaloGeom_;
+  edm::ESGetToken<GBRForestD, GBRDWrapperRcd> tokenRegressionKeyEB_;
+  edm::ESGetToken<GBRForestD, GBRDWrapperRcd> tokenUncertaintyKeyEB_;
+  edm::ESGetToken<GBRForestD, GBRDWrapperRcd> tokenRegressionReyEE_;
```

Comment:
Why "Rey" instead of "Key"?

Reply:
@perrotta I accidentally misspelled Key there. I will go back and change this in the code.